Makefile

```diff
@@ -20,7 +20,9 @@
 		metrics-unit-test docker-metrics-test
 
 # VERSION is the source revision that executables and images are built from.
-VERSION ?= $(shell git describe --tags --always --dirty || echo "unknown")
+#VERSION ?= $(shell git describe --tags --always --dirty || echo "unknown")
+# Above git describe is generating a wrong git tag
+VERSION ?= $(shell git describe --tags `git rev-list --tags --max-count=1` || echo "unknown")
```

Comment:
This removes the dirty flag which could be dangerous depending on how this version target is used. If a commit is not tagged with a version and a docker image is built using the version target to tag it, then you could overwrite a previous publish since this target is returning whatever the last tag was. 

I think the previous git command to retrieve the version is correct. It will append a short commit hash of the current commit if the current HEAD is not tagged and it will append `-dirty` if there are uncommitted changes. If HEAD is on a tag, you'll get the version with nothing appended: `v1.9.0`

Reply:
That's right, I will revert it back. Thank you!